config/environments/development.rb

```diff
@@ -39,7 +39,16 @@
 
   config.action_mailer.perform_caching = false
 
-  config.action_mailer.delivery_method = :letter_opener
+  config.action_mailer.delivery_method = :smtp
```

Comment:
We user sendgrid on production, not development. If letter_opener does't work for your task in development, disable letter_opener for time of test, and test your feature using server log

Reply:
If you look more attentively, you'll see, what i use sendgrid too (with my own settings, which you can find in google doc file). It works without any questions.